cassandra/delete_data.py

```diff
@@ -25,43 +30,71 @@ def __init__(self, refdes, uframe_ip, cassandra_ip_list):
+                protocol_version=3)
         self.session = cluster.connect('ooi')
-        stream_url = STREAM_METADATA_SERVICE_URL_TEMPLATE.format(uframe_ip)
-        partition_url = PARTITION_METADATA_SERVICE_URL_TEMPLATE.format(uframe_ip)
-        self.metadata_service_api = MetadataServiceAPI(stream_url, partition_url)
+        stream_url = STREAM_METADATA_SERVICE_URL_TEMPLATE.format(
+                uframe_ip)
+        partition_url = PARTITION_METADATA_SERVICE_URL_TEMPLATE.format(
+                uframe_ip)
+        self.metadata_service_api = MetadataServiceAPI(
+                stream_url, partition_url)
+        doi_url = DOI_SERVICE_URL_TEMPLATE.format(uframe_ip)
+        self.doi_service_api = DOIServiceAPI(doi_url)
 
     @staticmethod
     def parse_refdes(refdes):
         return refdes.split('-', 2)
 
     def get_stream_info(self):
         result = {}
-        partition_metadata_record_list = self.metadata_service_api.get_partition_metadata_records(
-            self.subsite, self.node, self.sensor
-        )
+        partition_metadata_record_list = (
+                self.metadata_service_api.get_partition_metadata_records(
+                        self.subsite, self.node, self.sensor))
         for partition_metadata_record in partition_metadata_record_list:
-            result.setdefault(partition_metadata_record['stream'], []).append(partition_metadata_record['bin'])
+            result.setdefault(
+                    partition_metadata_record['stream'], []
+                    ).append(partition_metadata_record['bin'])
         return result
 
     def delete_stream(self, stream, bins):
-        query = self.session.prepare('delete from %s where subsite=? and node=? and sensor=? and bin=?' % stream)
+        query = self.session.prepare(
+                '''delete from %s where subsite=? and node=? and sensor=? and
+                 bin=?''' % stream)
         for bin in bins:
-            self.session.execute(query, (self.subsite, self.node, self.sensor, bin))
+            self.session.execute(query, (
+                    self.subsite, self.node, self.sensor, bin))
 
     def delete_metadata(self):
-        self.metadata_service_api.delete_stream_metadata_records(self.subsite, self.node, self.sensor)
-        self.metadata_service_api.delete_partition_metadata_records(self.subsite, self.node, self.sensor)
+        self.metadata_service_api.delete_stream_metadata_records(
+                self.subsite, self.node, self.sensor)
+        self.metadata_service_api.delete_partition_metadata_records(
+                self.subsite, self.node, self.sensor)
 
     def delete_provenance(self):
-        query = self.session.prepare('delete from dataset_l0_provenance where subsite=? and node=? and sensor=?')
-        self.session.execute(query, (self.subsite, self.node, self.sensor))
+        query = self.session.prepare(
+                '''delete from dataset_l0_provenance where subsite=? and
```

Comment:
triple quotes in python indicates a comment block - this will result in prepare not receiving a parameter - whatever editor you are using is butchering the code.